packages/graph/Makefile.toml

```diff
@@ -18,31 +12,66 @@ command = "yarn"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
 
 [tasks.recreate-db]
+run_task = { name = ["recreate-db-task", "migrate-up"] }
+
+[tasks.recreate-db-task]
+private = true
 category = "Deploy"
 description = "Recreates the database"
-command = "yarn"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
+command = "yarn"
```

Comment:
Let's not do it in this PR but I think we should be extending the `tasks.yarn` task like we do for the `cargo` ones. We need to clean up this Makefile a bit, I've got some changes on a local branch to introduce the Makefile on another repo, can port them over.

Reply:
@Alfred-Mountfield Did we capture this in a ticket to `@todo`? (just wanted to check)